libs/application/templates/driving-license/src/forms/application.ts

```diff
@@ -29,12 +29,15 @@ export const application: Form = buildForm({
             buildRadioField({
               id: 'type',
               name: 'Tegund ökutækja',
-              emphasize: true,
+              largeButtons: true,
+              width: 'half',
               options: [
-                { value: 'car', label: 'Bifreiða- eða bifhjólaréttindi' },
+                { value: 'general', label: 'Almenn ökuréttindi' },
                 { value: 'truck', label: 'Vöru- eða hópbifreiðaréttindi' },
-                { value: 'tractor', label: 'Dráttarvélaréttindi' },
+                { value: 'bike', label: 'Bifhjólaréttindi' },
                 { value: 'taxi', label: 'Leigubílaréttindi' },
+                { value: 'tractor', label: 'Dráttarvélaréttindi' },
+                { value: 'trailer', label: 'Kerrur og eftirvagnar' },
```

Comment:
Are you not using the messages.ts for translatable names?

Reply:
Nope, not yet. Started like this to get familiar with the application system.